_includes/pbc/all/install-features/202307.0/install-glue-api/install-the-merchant-relationship-product-lists-glue-api.md

```diff
@@ -0,0 +1,124 @@
+
+
+## Install Merchant Relationship Product Lists Feature API
+
+### Prerequisites
+
+Install the required features:
+
+| NAME                          | VERSION          | INTEGRATION GUIDE                                                                                                                                                                       |
+|-------------------------------|------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Glue API: Spryker Core        | {{page.version}} | [Glue API: Spryker Core feature integration](/docs/pbc/all/miscellaneous/{{page.version}}/install-and-upgrade/install-glue-api/install-the-spryker-core-glue-api.html)                  |
+| Merchant Product Restrictions | {{page.version}} | [Merchant Product Restrictions feature integration](/docs/pbc/all/merchant-management/{{page.version}}/base-shop/install-and-upgrade/install-the-merchant-product-restrictions-feature.html)  |
+
+### 1) Install the required modules using Composer
+
+Run the following command to install the required modules:
+
+```bash
+composer require spryker/merchant-relationship-product-lists-rest-api:"^0.1.0" --update-with-dependencies
```

Comment:
see https://release.spryker.com/features/view/69

Reply:
Can you be a bit more specific please? It's the way how we already did for [Glue API - Category Management feature integration](https://docs.spryker.com/docs/pbc/all/product-information-management/202307.0/base-shop/install-and-upgrade/install-glue-api/install-the-category-management-glue-api.html). That's wrong?